config/locales/en/models/claim.yml

```diff
@@ -0,0 +1,26 @@
+# activerecord.errors.models.[model_name].attributes.[attribute_name]
+# activerecord.errors.models.[model_name]
+# activerecord.errors.messages
+# errors.attributes.[attribute_name]
+# errors.messages
+---
+
+en:
+  activerecord:
+    errors:
+      models:
+        claim/base_claim:
```

Comment:
this is what we were missing @naseberry  - the i18n-debug gem help to identify it